src/crontab.c

```diff
@@ -105,13 +110,16 @@ static void list_cmd(void),
 delete_cmd(void),
 edit_cmd(void),
 poke_daemon(void),
-check_error(const char *), parse_args(int c, char *v[]), die(int) ATTRIBUTE_NORETURN;
-static int replace_cmd(void), hostset_cmd(void), hostget_cmd(void);
-static char *host_specific_filename(const char *prefix, const char *suffix);
-static const char *tmp_path(void);
-
-static void usage(const char *msg) ATTRIBUTE_NORETURN;
-static void usage(const char *msg) {
+check_error(const char *), parse_args(int c, char *v[]),
+die(int) ATTRIBUTE_NORETURN;
+	 static int replace_cmd(void), hostset_cmd(void), hostget_cmd(void),
+	test_cmd(void), check_syntax(FILE *);
+	 static char *host_specific_filename(const char *prefix,
+	const char *suffix);
+	 static const char *tmp_path(void);
+
+	 static void usage(const char *msg) ATTRIBUTE_NORETURN;
+	 static void usage(const char *msg) {
```

Comment:
Please undo the indentation on these lines. They should just start at the beginning of line.